.github/workflows/linuxtests.yml

```diff
@@ -0,0 +1,50 @@
+name: Linux Tests
+
+on:
+  push:
+    branches:
+      - master
+      - maintenance/**
+  pull_request:
+    branches:
+      - master
+      - maintenance/**
+
+jobs:
+  Python-37-dbg:
```

Comment:
on the macOS testing script we have:

```
    if: "github.repository == 'scipy/scipy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
```
might be worth adding that here, so the run can be skipped if necessary

Reply:
ah yes will do